public/app/plugins/datasource/cloud-monitoring/datasource.ts

```diff
@@ -179,7 +179,18 @@ export default class CloudMonitoringDatasource extends DataSourceWithBackend<
           const dataQueryResponse = toDataQueryResponse({
             data: data,
           });
-          return dataQueryResponse?.data[0]?.meta?.custom?.labels ?? {};
+          return _map(dataQueryResponse?.data, 'meta.custom.labels')
```

Comment:
No need to import `map` from lodash - just use `dataQueryResponse?.data.map((f) => f.meta.custom.labels)`

Reply:
Thanks for advice, I tend to use lodash...
I fixed it in https://github.com/grafana/grafana/pull/44365/commits/5cf1cdf6a0a28b41ac882d345a67085002b6c6a0 .